hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/diskUsage/diskUsage.tsx

```diff
@@ -549,7 +546,7 @@ export class DiskUsage extends React.Component<Record<string, object>, IDUState>
                   </div>
                   <div className='dropdown-button'>
                     <Dropdown overlay={menu} placement='bottomCenter'>
-                      <Button>Display Limit: {(displayLimit === Number.MAX_VALUE) ? 'All' : displayLimit}</Button>
+                      <Button>Display Limit: {(displayLimit === Number.MAX_VALUE) ? MAX_DISPLAY_LIMIT : displayLimit}</Button>
```

Comment:
I think this has some from the previous code. Isn't this enough? 
```suggestion
                      <Button>Display Limit: {displayLimit}</Button>
```

Reply:
@dombizita 
Updated code in latest commit missed this comment earlier review.